flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/operators/join/stream/StreamingJoinOperatorTestBase.java

```diff
@@ -41,7 +41,7 @@
 /** Base test class for {@link AbstractStreamingJoinOperator}. */
 public abstract class StreamingJoinOperatorTestBase {
 
-    protected final InternalTypeInfo<RowData> leftTypeInfo =
+    protected InternalTypeInfo<RowData> leftTypeInfo =
```

Comment:
The final keyword shouldn't be remove for leftTypeInfo

Reply:
Yes, I changed this and only removed the final for rightType, rightKeySelector and assertor.